robot_skills/src/robot_skills/arm/gripper.py

```diff
@@ -0,0 +1,118 @@
+import rospy
+import PyKDL as kdl
+
+from actionlib import GoalStatus
+from robot_skills.robot_part import RobotPart
+from tue_manipulation_msgs.msg import GripperCommandGoal, GripperCommandAction
+from tue_msgs.msg import GripperCommand
+
+class GripperState(object):
+    """
+    Specifies a State either OPEN or CLOSE
+    """
+    OPEN = "open"
+    CLOSE = "close"
+
+class Gripper(RobotPart):
+    """
+    A gripper used for manipulating objects in the environment.
+    """
+    def __init__(self, robot_name, tf_listener):
+        """
+        constructor
+
+        :param robot_name: robot_name
```

Comment:
Again, not matching docstyle. Please check entire PR.

Reply:
This was a part of code that was moved. It was already there